swiftwinrt/Resources/Support/PropertyValue+ABI.swift

```diff
@@ -9,154 +9,164 @@ fileprivate var IID___x_ABI_CWindows_CFoundation_CIPropertyValueStatics: SUPPORT
 internal class IPropertyValueStatics: SUPPORT_MODULE.IInspectable {
       override public class var IID: SUPPORT_MODULE.IID { IID___x_ABI_CWindows_CFoundation_CIPropertyValueStatics }
 
-      internal func CreateUInt8Impl(_ value: UINT8) throws -> UnsafeMutablePointer<C_IInspectable>? {
-          var propertyValue: UnsafeMutablePointer<C_IInspectable>?
-          _ = try perform(as: __x_ABI_CWindows_CFoundation_CIPropertyValueStatics.self) { pThis in
-              try CHECKED(pThis.pointee.lpVtbl.pointee.CreateUInt8(pThis, value, &propertyValue))
+      internal func CreateUInt8Impl(_ value: UINT8) throws -> ComPtr<C_IInspectable>? {
```

Comment:
This now expresses double nullability since the ComPtr can hold null, and is itself optional.

Reply:
ComPtr can't really hold `null`. it only can as an optimization so that you can call `detach`.